unit-tests/test-phase.js

```diff
@@ -224,4 +223,19 @@ describe('phase', () => {
       assert.isFunction(phase.triggerGC);
     });
   });
+
+  describe('timeoutError', () => {
+    it('error thrown', () => {
+      let errorFunc = () => {
+        phase.timeoutError();
+      };
+      assert.throw(errorFunc, 'Test timeout exceeded ' + OPTIONS.timeout + 'ms');
```

Comment:
Nope:

  1 failing

  1) phase timeoutError error thrown:
     AssertionError: expected [Function] to throw error including 'Test timeout exceeded 30000ms' but got 'Cannot read property \'emit\' of undefined'